src/chrome/content/rules/Filippo.io.xml

```diff
@@ -1,10 +1,16 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://badfish.filippo.io/ => https://badfish.filippo.io/: (60, 'SSL certificate problem: self signed certificate in certificate chain')
+Fetch error: http://san.filippo.io/ => https://san.filippo.io/: (60, 'SSL certificate problem: self signed certificate')
+Fetch error: http://untrusted.filippo.io/ => https://untrusted.filippo.io/: (60, 'SSL certificate problem: self signed certificate in certificate chain')
+
```

Comment:
If they are invalid certificates why are we rewriting to them? I don't think there are users naively navigating to the HTTP version of untrusted.filippo.io who would like their connection upgraded. As I understand it, it is for a very specific testing use case, right?